spec/models/candidate_spec.rb

```diff
@@ -36,9 +36,9 @@
   context "with invalid attributes" do
     let(:invalid_attributes) do
        {
-        name: nil,
+        name: "john",
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.